trunk/source/TraceAbstraction/src/de/uni_freiburg/informatik/ultimate/plugins/generator/traceabstraction/BasicCegarLoop.java

```diff
@@ -683,7 +336,7 @@ assert accepts(getServices(), mInterpolAutomaton, mCounterexample.getWord(),
 				new IncrementalHoareTripleChecker(super.mCsToolkit, false)).getResult();
 	}
 
-	private static boolean
+	protected static boolean

```

Comment:
Is this really necessary as part of the CEGAR loop? 
It is also a quite expensive method....

Reply:
I'm absolutely not sure what the point of this method is. It's only used in `assert`s, but I don't know why this is even a desirable property. :confused: